docket.mk

```diff
@@ -1,4 +1,4 @@
-%_import : %.csv
+%_import : %.csv data
```

Comment:
Oh cool, I was afraid it might be more work to plug in the local files. I'll see if I can use this existing option and rework if I need to. Do you want me to do this in this PR or another?

Reply:
This PR would be great!